docs/src/tutorial.md

```diff
@@ -0,0 +1,157 @@
+# RangeEnclosures.jl basic tutorial
+
+This tutorial will teach you how to use RangeEnclosures.jl. First, we will give a basic overview of the package and its functionalities. Next, we will discuss in more details how to use the package in different scenarios (1D, higher dimension, using algorithms from external libraries etc.).
+
+## Setup
+
+Assuming you have [installed Julia](https://julialang.org/downloads/), you can install the package from the Julia REPL with the following lines.
+
+```julia
+using Pkg
+Pkg.add("RangeEnclosures.jl")
+```
+
+then you will need to import the package with
+
+```@example tutorial
+using RangeEnclosures
+```
+
+## Overview
+
+RangeEnclosures.jl is used to bound the range of o a given function. The main exported function is [`enclose`](@ref), and its basic usage is the following
```

Comment:
There are two different usages of the word "function" here. I do not have a good solution, but I suggest to add `f` to make the distinction a bit clearer.

```suggestion
RangeEnclosures is used to bound the range of a given function `f`. The main function provided by this package is [`enclose`](@ref), and its basic usage is the following.
```